### PR TITLE
[Improve] settings improvement

### PR DESCRIPTION
**Pull request by wolfboys**
[Improve] settings improvement

### CODE REVIEW

**Comment by wolfboys** [comment]
cc @GOODBOY008 PTAL, thx

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/bean/EmailConfig.java [hunk 1]
```diff
@@ -52,20 +43,32 @@ public class EmailConfig {
...
    * @return a new EmailConfig object that has its fields set according to the provided settings
    */
-  public static EmailConfig fromSetting(Map<String, Setting> settingMap) {
+  public static EmailConfig fromSetting() {
     try {
       EmailConfig emailConfig = new EmailConfig();
-      emailConfig.setSmtpHost(settingMap.get(KEY_ALERT_EMAIL_HOST).getSettingValue());
+
+      emailConfig.setSmtpHost(
+          SettingService.SETTINGS.get(SettingService.KEY_ALERT_EMAIL_HOST).getSettingValue());
```
Comment (GOODBOY008) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/bean/EmailConfig.java:53]: `SettingService` is not a bean. So `SETTINGS` always empty.

**Review by GOODBOY008 — APPROVED** [review]
LGTM

**Review by GOODBOY008 — APPROVED** [review]
LGTM

**Pull request merged**